client/signup/steps/site-topic/index.jsx

```diff
@@ -0,0 +1,99 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import StepWrapper from 'signup/step-wrapper';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormFieldset from 'components/forms/form-fieldset';
+
+class SiteTopicStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number.isRequired,
+		setSiteTopic: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	state = {
+		siteTopic: '',
+	};
+
+	onChangeTopic = event => this.setState( { siteTopic: event.target.value } );
+
+	// TODO:
+	// Handle submission.
+	submitSiteTopic( event ) {
+		event.preventDefault();
+	}
+
+	renderContent() {
+		const { translate } = this.props;
+		const { siteTopic } = this.state;
+
+		return (
+			<Card className="site-topic__content">
+				<form onSubmit={ this.submitSiteTopic }>
+					<FormFieldset>
+						<FormLabel htmlFor="siteTopic">{ translate( 'Type of Business' ) }</FormLabel>
+						<FormTextInput
+							id="siteTopic"
+							name="siteTopic"
+							placeholder={ translate( 'e.g. Fashion, travel, design, plumber, electrician' ) }
+							value={ this.state.siteTopic }
+							onChange={ this.onChangeTopic }
+							autoComplete="off"
+						/>
+					</FormFieldset>
+					<Button type="submit" disabled={ ! siteTopic } primary>
+						{ translate( 'Continue' ) }
+					</Button>
+					<span className="site-topic__form-description">
+						{ translate( 'Search above to continue' ) }
+					</span>
+				</form>
+			</Card>
+		);
+	}
+
+	render() {
+		const { translate } = this.props;
+		const headerText = translate( 'Search for your type of business.' );
+		const subHeaderText = translate( "Don't stress, you can change this later." );
+
+		return (
+			<div>
+				<StepWrapper
+					flowName={ this.props.flowName }
+					stepName={ this.props.stepName }
+					positionInFlow={ this.props.positionInFlow }
+					headerText={ headerText }
```

Comment:
@southp I believe we need the `fallbackHeaderText ` and `fallbackSubHeaderText` attributes here:
```
fallbackHeaderText={ headerText }
fallbackSubHeaderText={ subHeaderText }
```
`StepWrapper ` component takes the `headerText ` value only if this step is in the [first position ](https://github.com/Automattic/wp-calypso/blob/master/client/signup/step-wrapper/index.jsx#L67)in the flow. For all other positions, it uses only the `fallbackHeaderText` value. Same logic for `subHeaderText ` too. We will see the header and sub-header in /start/onboarding-dev/site-topic because onboarding-dev flow has only one step. However, this might not be the case when we merge this new step into the `onboarding` flow.